shapely/geos.py

```diff
@@ -74,7 +77,7 @@ def exists_conda_env():
         if len(geos_pyinstaller_so) == 1:
             _lgeos = CDLL(geos_pyinstaller_so[0])
             LOG.debug("Found GEOS DLL: %r, using it.", _lgeos)
-    elif exists_conda_env():
+    elif os.getenv('CONDA_PREFIX', ''):
```

Comment:
I don't know conda well enough to comment on this. Are there any docs/examples on why this is the best way? It looks sensible at least.

Reply:
This was changed from the CONDA_PREFIX to  `exists_conda_env` for a reason: https://github.com/Toblerity/Shapely/pull/843

@niklasarnitz what issue are you running into that you need to change this?